### PR TITLE
Feature/short navbar

### PR DESCRIPTION
**Pull request by dfruddffdc**
Shortened the header to only include the logo and a navbar.
This is a prerequisite to adding documentation pages which will need a sidebar, which would not leave much room for content under the really tall header.

Moved the old header content into a "hero image," only shown on the home page.

Responsive to different screen sizes. Screenshot below:
screen shot 2017-11-20 at 14 17 23

### CODE REVIEW

File: _sass/header.scss [hunk 1]
```diff
@@ -1,53 +1,39 @@
...
-    @media screen and (min-width: 64em) {
-        width: 275px;
-    }
-    @media screen and (min-width: 42em) and (max-width: 64em) {
-        width: 250px;
-    }
-    @media screen and (max-width: 42em) {
-        width: 200px;
+    * {
+        display: inline-flex;
```
Comment (pwrightffdc) [inline _sass/header.scss]: Same result can probably be achieved with `flex-direction: row;` set on the .navbar

Comment (dfruddffdc) [reply _sass/header.scss]: You are correct, I've updated the stylesheet.

**Comment by stephenwillcock** [comment]
The word "Orizuru" is rendering at a slightly different size from the menu items. Can we include this word in text so that it is guaranteed to match?
image

**Comment by dfruddffdc** [comment]
Code review action: Orizuru logo is now textual, for better alignment, and also can disappear on narrow screens.

Wideboy:
screen shot 2017-11-27 at 10 23 46

Skinny:
screen shot 2017-11-27 at 10 23 53

**Pull request merged**